pkgs/by-name/ra/rainbowcrack/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitLab
+, alglib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "rainbowcrack";
+    rev = "kali/${version}";
+    hash = "sha256-y6jdFVQSO80YLLOitPAY8TVMUsi58B6+3bFoncwr+Qo=";
+  };
+
+  buildInputs = [ alglib ];
+
+  installPhase = ''
+    runHook preinstall
+    mkdir -p $out/{bin,share/rainbowcrack}
+    cp -R *.txt $out/share/rainbowcrack
+    cp -R rt* rcrack $out/bin
+    chmod +x $out/bin/*
+    runHook postinstall
+  '';
+
+  meta = with lib; {
```

Comment:
`license` must be set

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

```
    If there is no upstream license, meta.license should default to lib.licenses.unfree.
    If in doubt, try to contact the upstream developers for clarification.
```

Reply:
I checked other Distro like Kali and Debian and the license is unset but like advised i will also reach out to upstream developers for clarification.

Reply:
as per the above link `license` must be set in nixpkgs

we can set it to  `licenses.unfree` and update in other PR if we hear back from upstream